tests/CounterInShopingBag.spec.ts

```diff
@@ -0,0 +1,82 @@
+import {test, expect} from '@playwright/test'
+import {NavigationPage } from '../page-objects/navigationPage'
+import {BeddingPage} from '../page-objects/beddingPage'
+
+test.beforeEach(async ({page})=>{
+    
+    await page.goto('https://www.shopist.io/')
+
+})
+    
+    test('Counter in shopping bag', async ({ page }) => {
+        
+        const navigateTo = new NavigationPage(page);
+        const onBeddingPage = new BeddingPage(page);
+
+        //navigate to Shop Now page 
+        await navigateTo.shopNowPage()
+
+        //navigate to Bedding page
+        await navigateTo.beddingPage()
+
+        //Click on the White Linen Duvet Cover 
+        await onBeddingPage.clickOnWhiteLinenDuvetCover()
+
+        //price before cart page 
+        const initialPrice = await page.getByText('$').innerText()
+
+        // Removing the $ sign
+        const priceWithout$sign = initialPrice.replace(/[^\d.]/g, '');
```

Comment:
It is not good practice to put any sign or special character in the variable's name. 
You can write 'currency' or 'dollar' instead of $.